Documentation/migration-guide.md

```diff
@@ -0,0 +1,244 @@
+## Migrating from Media to EdgeMedia
+
+This is the complete migration guide from Media to EdgeMedia SDK.
+
+| Quick Links |
+| --- |
+| [Configuration](#configuration)  |
+| [Add extensions to your app](#add-the-edgemedia-extension-to-your-app) <ul> <li>[Dependencies](#dependencies)<li> [Download extension with dependencies](#download-extension-with-dependencies) <li> [Import and register extensions](#import-and-register-extensions) </ul> |
+| [Granular ad tracking](#granular-ad-tracking)  |
+| [Downloaded content tracking](#downloaded-content-tracking)  |
+| [API Reference](#api-reference)|
+
+------
+
+## Configuration 
+
+### Media
+| Name | Key | Value | Required |
+| --- | --- | --- | --- |
+| Collection API Server | "media.trackingServer" | String | Yes |
+| Channel | "media.channel" | String | No |
+| Player Name | "media.playerName" | String | No |
+| Application Version | "media.appVersion" | String | No |
+
+### EdgeMedia
```

Comment:
Use "Media for Edge Network" throughout this file when referring to the extension.